components/clm/bld/namelist_files/use_cases/2010_CMIP6_control.xml

```diff
@@ -4,22 +4,17 @@
 
 <use_case_desc>Conditions to simulate 2010 land-use</use_case_desc>
 
-<sim_year>2015</sim_year>
+<sim_year>2010</sim_year>
 
 <sim_year_range>constant</sim_year_range>
 
-<stream_year_first_ndep phys="clm4_5" use_cn=".true." ndepsrc="stream" >2015</stream_year_first_ndep>
-<stream_year_last_ndep  phys="clm4_5" use_cn=".true." ndepsrc="stream" >2015</stream_year_last_ndep>
+<stream_year_first_ndep phys="clm4_5" use_cn=".true." ndepsrc="stream" >2010</stream_year_first_ndep>
+<stream_year_last_ndep  phys="clm4_5" use_cn=".true." ndepsrc="stream" >2010</stream_year_last_ndep>
 
-<stream_year_first_pdep phys="clm4_5" use_cn=".true." ndepsrc="stream" >2015</stream_year_first_pdep>
-<stream_year_last_pdep  phys="clm4_5" use_cn=".true." ndepsrc="stream" >2015</stream_year_last_pdep>
+<stream_year_first_pdep phys="clm4_5" use_cn=".true." ndepsrc="stream" >2010</stream_year_first_pdep>
+<stream_year_last_pdep  phys="clm4_5" use_cn=".true." ndepsrc="stream" >2010</stream_year_last_pdep>
 
-<stream_year_first_popdens phys="clm4_5" use_cn=".true." ndepsrc="stream" >2015</stream_year_first_popdens>
-<stream_year_last_popdens  phys="clm4_5" use_cn=".true." ndepsrc="stream" >2015</stream_year_last_popdens>
-
-<!-- CMIP6 DECK compsets -->
-<fsurdat hgrid="ne30np4" >lnd/clm2/surfdata_map/surfdata_ne30np4_simyr2010_c20181025.nc </fsurdat>
-<fsurdat hgrid="r0125" >lnd/clm2/surfdata_map/surfdata_0.125x0.125_simyr2010_c191025.nc </fsurdat>
-<finidat hgrid="ne30np4" >lnd/clm2/initdata_map/20180716.DECKv1b_A3.ne30_oEC.edison.clm2.r.2010-01-01-00000.nc </finidat>
```

Comment:
By removing this, there would be no default finidat for ne30np4 when not using tri-grid for F2010SC5-CMIP6. Do we want to keep this change?

Reply:
It appears making the change or not does not matter. Either with the master or this branch, when configuring to use ne30np4 for land, this finidat file would not be picked up. Furthermore, When manually specifying the file in user_nl_clm, the run would fail during lnd initialization.

A new land IC needs to be created anyway if not to use a null finidat, and should be added to namelist_defaults_clm4_5.xml instead of this generic use_case file for compset F2010SC5-CMIP6.

Reply:
@tangq, @wlin7, Shouldn't the finidat entry for ne30np4 be added in the namelist_defaults_clm4_5.xml?

Reply:
@bishtgautam I did intend to do just that. But tests using master or the branch for this PR can't run with the ne30np4 finidat file as shown above. We need a working file first, which can be done separately.

Reply:
Does the model exits when `20180716.DECKv1b_A3.ne30_oEC.edison.clm2.r.2010-01-01-00000.nc` is used?